tools/eslint-rules/only-ascii-characters.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @fileoverview Prohibit the use of non-ascii characters
+ * @author Kalon Hinds
+ */
+
+/* eslint no-control-regex:0 */
```

Comment:
I'd prefer see `eslint-disable-line` or `eslint-disable-next-line` to target the places where the control characters are needed.